src/main/java/uk/gov/hmcts/darts/task/runner/impl/ProcessDetsToArmResponseAutomatedTask.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.darts.task.runner.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.arm.component.AutomatedTaskProcessorFactory;
+import uk.gov.hmcts.darts.common.repository.AutomatedTaskRepository;
+import uk.gov.hmcts.darts.log.api.LogApi;
+import uk.gov.hmcts.darts.task.api.AutomatedTaskName;
+import uk.gov.hmcts.darts.task.config.AutomatedTaskConfigurationProperties;
+import uk.gov.hmcts.darts.task.runner.AutoloadingManualTask;
```

Comment:
I'd suggest we also add a Processor class with an empty "execute" method so we can tackle more of the scaffolding stuff as part of this ticket, which is the whole point we usually raise a dedicated scaffolding ticket for automated tasks.
We have a PR that we usually use as reference here: https://github.com/hmcts/darts-api/pull/1559/files
Sadly it's a bit out of date now but still can give you an idea

Reply:
I think we are moving away from the processor approach as it does not make much sense abstracting this logic out again